### ISSUE TITLE
FI-008

### ISSUE THREAD

**Issue by mattgarrish** [daisy/epub-accessibility-iso#32]
Many relevant terms and definitions are missing.

Proposed Change: Add definitions to e.g. accessibility,  accessibility metadata and assistive   technology. 

According to ISO 5127, accessibility is “usability of a product, service (3.1.1.59), environment or facility by people with the widest range of capabilities” (ISO 5127 definition 3.11.1.02).

DCMI definition for accessibility metadata is “metadata that describes the accessibility of resources and services, usually those on or available through, the web”. (https://www.dublincore.org/groups/access/standards/)

(Clause 3)

**Comment by mattgarrish** [comment]
Accepted

Actions:
- We borrow two definitions and introduce in clause 3 (accessibility and Accessibility metadata) with caveat that we modify the original definitions and point to original definitions in ISO.
- Definitions of Author, Reading System, Assistive Technology should be copied to this document and reference to EPUB 3 as source of definitions should be provided.
- Instead of using “package document” we should use “EPUB package document”

**Comment by mattgarrish** [comment]
I'm not sure how this solution will work. If we bring in the definitions from EPUB, then we end up with references to a whole lot of other definitions, as the source definitions use a lot of EPUB-specific terms and delve into concepts unrelated to this specification.

The original accessibility specification defined generic versions of Author, Reading System, EPUB Publication and Package Document to largely avoid this problem (i.e, only Rendition also needed to be included). Should we go back and re-incorporate those definitions?

/cc @avneeshsingh @GeorgeKerscher @murata2makoto

**Comment by GeorgeKerscher** [comment]
Sorry, no opinion here. What was said is that if there is a word used somewhere it should be defined so that one could replace the word with the definition in context.

**Comment by murata2makoto** [comment]
If you borrow just one term from a spec, you might want to copy the definition in your terminology section and indicate the source.  But if you borrow several terms from a spec (e.g., EPUB), you only have to normatively reference EPUB without providing definitions of terms.

**Comment by mattgarrish** [comment]
> But if you borrow several terms from a spec (e.g., EPUB), you only have to normatively reference EPUB without providing definitions of terms.

This is what we did after the earlier meeting in Milan. We dropped the definitions from the Accessibility specification and replaced them with this sentence in the terminology:

> This document also uses the terms Author, EPUB Content Document, EPUB Publication, EPUB Package Document, and Reading System as defined in [ISO 23736-2].

But now this resolution says to add in the actual definitions. Could we perhaps just partially accept this and note that we already include a reference to where the EPUB definitions are defined so there's no need to include the actual definitions?

**Comment by mattgarrish** [comment]
Looking at the template document, I suppose the introductory sentence should be changed to:

> For the purposes of this document, the terms and definitions given in [ISO 23736-2], [WCAG 2.0] and the following apply.

Do we then omit the specific list of terms we're using? Is that where the confusion may have arisen - that they didn't look at these lists we'd specified from EPUB and WCAG?

**Comment by murata2makoto** [comment]
At the beginning of Clause 3, we should not reference any specifications.  I received the same comment from ISO for a different DIS.  If a spec is normatively referenced, they take it for granted that all terms in that spec are borrowed.  Please do not touch the introductory sentence.
